layouts/partials/css.html

```diff
@@ -19,6 +19,9 @@
 <link rel="stylesheet" href="{{ "css/jquery-ui.min.css" | relURL }}">
 <link rel="stylesheet" href="{{ "css/callouts.css" | relURL }}">
 <link rel="stylesheet" href="{{ "css/custom-jekyll/tags.css" | relURL }}">
+{{ if eq .Params.mermaid true }}
+<script async src="https://unpkg.com/mermaid@8.5.0/dist/mermaid.min.js"></script>
```

Comment:
When this goes live, does it make sense to copy this into the built docs so it goes onto the Netlify CDN?

Reply:
I like the idea of reducing external dependencies but I do want to make this very low-touch to support. Let me try a couple things...